deps.edn

```diff
@@ -139,7 +134,8 @@
    ;; - WSM 2021-09-13
    :main-opts  ["-m" "eastwood.lint"
                 {:source-paths ["src" "src-docs"]
-                 :test-paths   ["test"]}]}
+                 ;; :test-paths   ["test"]
```

Comment:
Looks like we can delete this line now

Reply:
I just did this to get the CI to go green - it throws a very weird error that I couldn't figure out how to solve.

Are we ok with not using Eastwood to lint test-paths? Linting test code isn't as critical as source code, but it can still catch some issues. I'd be happy to remove this if we're ok with that, otherwise I wanted to keep it as an implicit TODO to figure out how to fix it. In which case I should make explicit : )

Reply:
Hmm... it works fine locally (with just `make eastwood`) for me even w/ the test-paths enabled.

Reply:
It also works for me when run as `script/run-in-docker.sh make ci` w/ test-paths enabled. That runs in Docker so I'm not sure what could be different there. Are you seeing errors in GitHub Actions or on your local machine (or both)?

Reply:
I just checked and it's only on my local machine - the pipeline failed so I ran it locally and it blew up, and I never verified it was the same error as the pipeline : (

The linting stays on :+1: